services/minespace-web/src/components/pages/Incidents/IncidentGetStarted.js

```diff
@@ -11,26 +11,42 @@ export const IncidentGetStarted = () => {
           Verbal notice of a reportable incident, within 4 hours
         </Typography.Title>
         <Typography.Paragraph>
-          Reportable incidents that cause <b>serious injury</b> or <b>loss of life</b> must be{" "}
-          <b>reported verbally to the Ministry within 4 hours</b> of the incident occurrence. (Mines
-          code, section 1.7.1(1)(a))
+          Reportable incidents that cause 
+          {' '}
+          <b>serious injury</b>
+          {' '}
+or 
+          {' '}
+          <b>loss of life</b>
+          {' '}
+must be
+          {" "}
+          <b>reported verbally to the Ministry within 4 hours</b>
+          {' '}
+of the incident occurrence. (HSRC
+          1.7.2)
         </Typography.Paragraph>
         <Typography.Paragraph>
-          Call the <b>Mine Incident Reporting Line at </b>
+          Call the 
+          {' '}
+          <b>Mine Incident Reporting Line at </b>
           <a href="tel:1-888-348-0299">
             <b>1-888-348-0299</b>
-          </a>{" "}
+          </a>
+          {" "}
           . This line is monitored 24/7 by an On-call Inspector.
```

Comment:
this looks like there's a space before the period, is that how it renders?

Reply:
Ack, damn IDE